content/concepts/the-rippled-server/peer-protocol.md

```diff
@@ -17,152 +26,34 @@ ip = 0.0.0.0
-
-#### Request Format
-
-To request the Peer Crawler information, make the following HTTP request:
-
-* **Protocol:** https
-* **HTTP Method:** GET
-* **Host:** (any `rippled` server, by hostname or IP address)
-* **Port:** (port number where the `rippled` server uses the Peer Protocol, typically 51235)
-* **Path:** `/crawl`
-* **Notes:** Most `rippled` servers use a self-signed certificate to respond to the request. By default, most tools (including web browsers) flag or block such responses for being untrusted. You must ignore the certificate checking (for example, if using cURL, add the `--insecure` flag) to display a response from those servers.
-
-#### Response Format
-
-The response has the status code **200 OK** and a JSON object in the message body.
-
-The JSON object has the following fields:
-
-| `Field`          | Value | Description                                       |
-|:-----------------|:------|:--------------------------------------------------|
-| `overlay.active` | Array | Array of Peer Objects, where each member is a peer that is connected to this server. |
+The peer protocol port also serves the [special Peer Crawler API method](peer-crawler.html).
 
-Each member of the `active` array is a Peer Object with the following fields:
+### Node Key Pair
 
-| `Field`      | Value                    | Description                        |
-|:-------------|:-------------------------|:-----------------------------------|
-| `ip`         | String (IPv4 Address)    | The IP address of this connected peer. |
-| `port`       | String (Number)          | The port number on the peer server that serves RTXP. Typically 51235. |
-| `public_key` | String (Base-64 Encoded) | The public key of the ECDSA key pair used by this peer to sign RTXP messages. (This is the same data as the `pubkey_node` reported in the peer server's [`server_info` command](server_info.html).) |
-| `type`       | String                   | The value `in` or `out`, indicating whether the TCP connection to the peer is incoming or outgoing. |
-| `uptime`     | Number                   | The number of seconds the server has been connected to this peer. |
-| `version`    | String                   | The `rippled` version number the peer reports to be using. |
+When a server first starts up, it generates a _node key pair_ to use to identify itself in peer protocol communications. The server uses its key to sign all its peer protocol communications. This makes it possible to reliably identify and verify the integrity of messages from another server in the peer-to-peer network even if that server's messages are being relayed by untrusted peers. A node key pair is similar to a validator's ephemeral key pair, but validation key pairs are treated separately from node key pairs.
```

Comment:
> A node key pair is similar to a validator's ephemeral key pair, but validation key pairs are treated separately from node key pairs.

Can you explain why they are treated differently? Might be able to remove that line and have the paragraph still make sense.